lumicks/pylake/force_calibration/calibration_models.py

```diff
@@ -11,16 +13,16 @@ def __init__(self, bead_diameter):
     def __name__(self):
         raise NotImplementedError
 
-    def __call__(self):
-        raise NotImplementedError
+    def __call__(self, f, fc, diffusion_constant, f_diode, alpha):
+        return passive_power_spectrum_model(f, fc, diffusion_constant, f_diode, alpha)
```

Comment:
What do you mean by temporary thing?

This call is used by the spectral fitter to fit the thermal calibration model. The thermal calibration part is the same for active and passive calibration.  Eventually, there are quite a few extra flags going into that model. That'd mean that the amount of duplication would increase over time. I'd prefer to have it here, because eventually, you'd end up duplicating more than just this.